### PR TITLE
Dropdown to switch view of company kanban cards based on city team

### PR DESCRIPTION
**Pull request by KNWR**
Trello ticket 117 - adding a dropdown to pipeline view so that partners can switch between viewing companies associated with one particular team or another

Screen Shot 2019-12-26 at 9 48 37 PM

Note - thought about implementing w/composition (specialization), since  PartnerDropdown and PartnerTeam Dropdown are largely the same component w/diff dropdown elements (fed via diff props) and callback fn's but seemed a little involved w/static typing & the props.

(Looked at: https://www.pluralsight.com/guides/composing-react-components-with-typescript)

### CODE REVIEW

**Comment by kirubarajan** [comment]
Is it possible to leave a quick comment at the top of `client/components/Pipeline/PartnerTeamDropdown.tsx`? Other than that, LGTM.

File: client/components/Pipeline/Kanban.tsx [hunk 1]
```diff
@@ -178,13 +180,22 @@ export default class Kanban extends PureComponent<KanbanProps, KanbanState> {
...
-          <CustomDropdown partners={this.state.partnerNames} />
+          <PartnerDropdown partners={this.state.partnerNames} />
           <IndividualButton
             loggedInPartnerName={`${this.props.user.firstName} ${
               this.props.user.lastName
             }`}
           />
           <GroupButton />
+          <PartnerTeamDropdown partnerTeams={Object.keys(Team)} />
+          {/* <IndividualButton
```
Comment (KNWR) [inline client/components/Pipeline/Kanban.tsx]: I should delete the commented out stuff here

Comment (VinaiRachakonda) [reply client/components/Pipeline/Kanban.tsx]: Yea can you delete

**Review by VinaiRachakonda — APPROVED** [review]
This is hot

**Pull request merged**